src/extension.ts

```diff
@@ -7,29 +7,32 @@ import { version, displayName } from "../package.json";
 import { TestRunner } from "./api/runner";
 import { createChildProcess, createTestplaneMasterRpc } from "./api";
 import { registerCommands } from "./commands";
+import { registerViews } from "./views";
 import { CONFIG_GLOB } from "./constants";
 
 const CONFIG_DEBOUNCE_WAIT = 300;
 
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
     // eslint-disable-next-line no-use-before-define
-    const extension = new TestplaneExtension();
+    const extension = new TestplaneExtension(context);
```

Comment:
I need to send this context inside my settings webview in order to use workspace storage and make correct path to assets (css and js file).